app/Contracts/PhotoCreate/PhotoDTO.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Contracts\PhotoCreate;
+
+use App\Models\Photo;
+
+interface PhotoDTO
```

Comment:
Should this file be in DTO instead of contracts?

Reply:
It is an interface, hence a contract.
I am open to another other name.

Reply:
Sure. Only because the others were in the DTO folder. If this makes more sense to you I'm happy as-is.

Reply:
Could consider `PhotoDTOInterface` or `HasGetPhoto`. :)

Reply:
Could do - I think the current name is nice. Let's just leave it as-is.